sql/telemetry_derived/clients_first_seen_v1/query.sql

```diff
@@ -0,0 +1,25 @@
+WITH today AS (
```

Comment:
If we productionize this, I would add some information at the top here.

This query is weird in that we'd be rewriting the entire table every day rather than a single partition. Most rows will stay the same, but some will get second_seen_date filled in.

I believe the full size of this table will only be ~2 TB, so rewriting that amount of data every day seems reasonable. The `init.sql` query will have to scan the full history of clients_daily, so several hundred TB. It's expensive in dollars, but it appears to run quite quickly. It took ~1 minute when run for a single sample_id.